### DISCUSSION TITLE
Adding Event to Calendar via Deeplink not functional with Office 365 URL

### DISCUSSION THREAD

**Discussion by no3rdw** [InteractionDesignFoundation/add-event-to-calendar-docs#63] [General]
The deeplink described [here](https://interactiondesignfoundation.github.io/add-event-to-calendar-docs/services/outlook-web.html) used to work some months ago using the outlook.office.com domain (described in the linked doc), but now throws a javascript error and does not complete the action to add the event to the user's O365 calendar.

Clicking the 'Add a test event' link at the top of the doc above works using outlook.live.com, but replacing the domain with outlook.office.com does not work.

**Comment by alies-dev** [comment]
Partial result while re-testing the Outlook documentation.

`outlook.live.com` works today. This link opens the compose form with every field populated, verified on 2026-08-19:

```
https://outlook.live.com/calendar/0/deeplink/compose?path=/calendar/action/compose&rru=addevent&startdt=2026-12-31T19:30:00Z&enddt=2026-12-31T22:30:00Z&subject=Birthday&body=With%20clowns%20and%20stuff&location=North%20Pole
```

The same URL against `outlook.office.com` could not be tested here: it redirects to `login.microsoftonline.com` and needs a work or school account, which is not available in this environment.

One thing that did come out of the attempt: the sign-in redirect preserves the deep link. The full compose URL is carried through the OAuth `state` parameter and comes back after authentication, so the parameters are not lost on the login round trip. Whatever breaks on Office 365 happens after that, in the application itself.

Leaving open. Anyone with an Office 365 mailbox: the useful detail to report is the exact JavaScript error from the console, plus whether the compose form appears at all or the page stays blank.

**Reply by no3rdw** [reply]
Hi! I can confirm that the office365 deeplink example in the docs (and below) does work as of today, with event title, location, start and end datetimes, and body populated.

`https://outlook.office.com/calendar/deeplink/compose?path=/calendar/action/compose&rru=addevent&startdt=2023-08-09T19:30:00Z&enddt=2023-08-09T22:30:00Z&subject=Birthday&body=With%20clowns%20and%20stuff&location=North%20Pole`

<img width="1263" height="816" alt="Screenshot 2026-08-19 093625" src="https://github.com/user-attachments/assets/aa5022db-3b65-4713-aef8-4189859a8d44" />

I'm not sure when this started working again as it's been quite a while since I saw the issue - and don't remember what the javascript error was at the time!  But it seems to no longer be an issue :)
